src/main/java/shef/data/ApiExample.java

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
What does this class do exactly? I'm having trouble understanding it.

Reply:
Deciding to put this in next PR.